api/src/main/java/org/openmrs/util/OpenmrsUtil.java

```diff
@@ -979,11 +979,9 @@ public static boolean deleteDirectory(File dir) throws IOException {
 				deleteDirectory(f);
 			}
 			boolean success = f.delete();
-			
 			if (log.isDebugEnabled()) {
 				log.debug("   deleting " + f.getName() + " : " + (success ? "ok" : "failed"));
 			}
-			
 			if (!success) {
 				f.deleteOnExit();
 			}
```

Comment:
@gracebish thanks for the work
are there anychanges needed in this class ie` OpenmrsUtil` ,

Reply:
@gitcliff  Any change on the file causes errors which also breaks mvn. Kindly advise on how to deal with this.

Reply:
you can manually fix them by adding back the deleted lines